### ISSUE TITLE
Implement simple means of dialogue

### ISSUE THREAD

**Issue by oliver-ruehl** [Hirnbix/captain-holetooth#41]
Keep in mind:

- Simple so that i can understand this
- Very simple branching of conversation
- Suitable for younger players (I'm thinking of icons like "yes" = "head nodding animation PNG" 
- Maybe possibility to add simple icons like mentioned above

DRAFT, need to go :(

**Comment by oliver-ruehl** [comment]
Is this something we can use or build up?
https://github.com/brunosxs/SMRT-Godot

**Comment by oliver-ruehl** [comment]
I've implemented something.
Right now only one line of text works, but that should be fine for kids

**Comment by shelltitan** [comment]
You can use a separate scene for a dialog tree and instance it when the dialogue starts.

**Comment by oliver-ruehl** [comment]
Yes, I want it to be more flexible so that I can call it from any NPC and NOT have it attached to each NPC.
